packages/compass-indexes/src/modules/search-indexes.ts

```diff
@@ -1,5 +1,20 @@
 import type { AnyAction } from 'redux';
+import { createLoggerAndTelemetry } from '@mongodb-js/compass-logging';
 import { isAction } from './../utils/is-action';
+import type { IndexesThunkAction } from '.';
+
+import type { SearchIndex } from 'mongodb-data-service';
+
+const { debug } = createLoggerAndTelemetry('COMPASS-INDEXES');
+
+export type SearchSortColumn = keyof typeof sortColumnToProps;
+export type SortDirection = 'asc' | 'desc';
```

Comment:
We now have multiple of these. Trying to think of where to put the SortDirection type and if I should bother. It is just 'asc' | 'desc' everywhere anyway.

Reply:
Maybe where we have `RootState` defined?